regulations/static/regulations/js/source/views/header/preamble-head-view.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
Just as an aside, we don't *have* to make an extra view here. The `PreambleView` *can* reach up into the DOM to find elements in the header--which is effectively what happens when the `PreambleView` instantiates a `PreambleHeadView` anyway. I think the value in having smaller views is the potential for reuse--we might eventually rename `PreambleHeadView` to `CommentHeadView` and re-use it when we introduce commenting on diffs, for example.